packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -98,7 +98,7 @@ function InteractionLayer ({
       activeTool.deleteMark(activeMark)
       setActiveMark(undefined)
     } else {
-      activeMark.setSubTaskVisibility(true, node)
+      activeMark?.setSubTaskVisibility(true, node)
```

Comment:
I'm not sure why, but `activeMark` is undefined here in the stories.

Reply:
Figured it out: the first Transcription Line story calls `transcriptionLine.finish()` when the line isn't active. In a real drawing task, a mark is always active when receiving events, so only active marks can be finished.

Reply:
Fixed by adding controls to each story, but the existence check may still be useful.